### ISSUE TITLE
Keras tuner terminates at the end of Epoch 1

### ISSUE THREAD

**Issue by UtpalM** [keras-team/keras-tuner#400]
I am using the RandomSearch tuner in Keras but it exits at the end of Epoch 1. Please see attached stack trace (with error information at the end) for review.  I am not sure what I need to change on my end. Appreciate any insight. Thanks!

![Capture11](https://user-images.githubusercontent.com/30846264/92156953-b655c200-edf7-11ea-977e-6a6b838ce8dd.JPG)

![Epoch1](https://user-images.githubusercontent.com/30846264/92157057-e3a27000-edf7-11ea-90de-ff2c39e8efe6.JPG)

![Err1](https://user-images.githubusercontent.com/30846264/92156759-77c00780-edf7-11ea-8605-6edf642deb8a.JPG)

![Err2](https://user-images.githubusercontent.com/30846264/92156766-78589e00-edf7-11ea-85f8-e464ccd41bb9.JPG)

**Comment by UtpalMattoo** [comment]
I will try to follow this post to see if these ideas work for me:

**Comment by yixingfu** [comment]
@UtpalMattoo Did it work? I would expect this problem to be solved for master branch

**Comment by UtpalMattoo** [comment]
@yixingfu No, it didn't. I am not sure why it's stopping at the end of epoch 1. 
Pl. see the TPU set up, end of epoch 1 and the third screen save where the messages from epoch 1 are cleared out but the epoch 2 does not start. But - I did not get the error that I got in the first post

![TPUSetUp](https://user-images.githubusercontent.com/30846264/92313952-7d585180-ef9f-11ea-8dd9-21f0905fe4bd.JPG)
![EndOfEpoch1](https://user-images.githubusercontent.com/30846264/92313955-82b59c00-ef9f-11ea-94f8-1467fc4c3662.JPG)
![Epoch2DoesNotStart](https://user-images.githubusercontent.com/30846264/92313956-8517f600-ef9f-11ea-9eeb-18a74e8b4ada.JPG)

**Comment by yixingfu** [comment]
@UtpalMattoo from the print out I think you are using the latest release; however the fix is currently only available with master branch (installed with `pip install git+https://github.com/keras-team/keras-tuner`. You may need to restart the session.)

**Comment by UtpalMattoo** [comment]
@yixingfu - see the keras tuner I am using. And also how the keras tuner is set up as `pip install -U keras-tuner`.

Do you mean the keras tuner should instead be set up as `pip install git+https://github.com/keras-team/keras-tuner` for the fix to take effect? (see the keras tuner set-up in my workbook at the end included as a screenshot)

Second - I don't understand when you say - 

> the latest release which is a bit dated right now

If it is latest, how is it dated?

![KerasTunerInUse](https://user-images.githubusercontent.com/30846264/92314099-5ac73800-efa1-11ea-9301-2783a40fba20.JPG)

**Comment by yixingfu** [comment]
@UtpalMattoo The latest release is 30/oct/2019, and there has been a lot of changes since then, including fixing this problem. Using the line I mentioned to install from github is not ideal, but a temporary workaround before the next release comes out.

**Comment by UtpalMattoo** [comment]
@yixingfu - Thanks. I will use the line `pip install git+https://github.com/keras-team/keras-tuner` and see if that moves this along to epoch 2 and more if I need to

**Comment by yixingfu** [comment]
Also, looks like you are working on Kaggle, so allow me shamelessly advertise my [notebook](https://www.kaggle.com/fuyixing/starter-keras-tuner-efficientnet-tpu) showing how to use this.

**Comment by UtpalMattoo** [comment]
That's great. Thanks for sharing. Yes, I am working on Kaggle. 
Now I can follow your work there....
